server/src/datastore/sql/migrations/001-initial.sql

```diff
@@ -15,3 +15,10 @@ CREATE TABLE posts (
   postedAt INTEGER NOT NULL,
   FOREIGN KEY (userId) REFERENCES users (id)
 );
+
+CREATE TABLE likes (
```

Comment:
please add a new migration file with these changes, something like `002-add-likes`.
the point of database migrations is that they're incremental, each change is isolated in a file so that the migration library can decide which changes don't exist in the db yet and need to be run.